os/arch/arm/src/artik053/src/artik053_boot.c

```diff
@@ -150,6 +200,16 @@ static void board_audio_initialize(void)
 #endif
 }
 
+static void board_wdt_initialize(void)
+{
+#ifdef CONFIG_S5J_WATCHDOG
+#ifdef CONFIG_WATCHDOG
```

Comment:
1. No watchdog is not needed for audio. Earlier we thought watchdog timer is used by audio device driver, and since we ported the new watchdog timer , I thought to push it into this pull request.
2. This can be done , but the Travis build config files contain CONFIG_WATCHDOG=y and CONFIG_S5J_WATCHDOG=n or not defined. . This combination in config files is incorrect and that is why I added S5J_WATCHDOG dependency wherever it was needed to have the github build pass. This is anyways meant to be a temporary solution.